database/seeds/SourceServiceTableSeeder.php

```diff
@@ -319,6 +364,33 @@ private function addArkVariables()
         ]);
     }
 
+    private function addCSGOVariables()
+    {
+        ServiceVariable::updateOrCreate([
+            'option_id' => $this->option['csgo']->id,
+            'env_variable' => 'SRCDS_MAP',
+        ], [
+            'name' => 'Map',
+            'description' => 'The default map for the server.',
+            'default_value' => '',
+            'user_viewable' => 1,
+            'user_editable' => 1,
+            'rules' => 'required',
+        ]);
+
+        ServiceVariable::updateOrCreate([
+            'option_id' => $this->option['csgo']->id,
+            'env_variable' => 'STEAM_ACC',
+        ], [
+            'name' => 'Steam Account Token',
+            'description' => 'The Steam Account Token required for the server to be displayed publicly.',
+            'default_value' => '',
+            'user_viewable' => 1,
+            'user_editable' => 1,
+            'rules' => 'required',
```

Comment:
Rule should probably be `required|string|alpha_num|size:32` since those token lengths don't change as far as I am aware. Check the size to be sure what the number of characters is though.